shell.nix

```diff
@@ -3,4 +3,8 @@
 }:
 pkgs.mkShell {
   buildInputs = pkgs.lib.attrsets.mapAttrsToList (name: value: value) default.toolAttrs;
+
+  shellHook = ''
+    pre-commit install
```

Comment:
Does this mean this is not optional?

Reply:
Yes, but you can now opt-out by setting `DADE_NO_PRE_COMMIT` e.g. in your `.envrc.private`.

Reply:
Thanks!